### PR TITLE
Fix problems keeping sim from being usable

### PR DESCRIPTION
**Pull request by DrYerzinia**
Fixes things Jeff broke

### CODE REVIEW

File: cusub_cortex/cusub_cortex_bringup/launch/cortex.launch [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
   <group ns="leviathan">
 
+    <arg name="config_filename" default="mission_config.yaml"/>
+
     <include file="$(find mapper)/launch/mapper.launch"/>
+
+    <include file="$(find waypoint_navigator)/launch/waypoint_navigator.launch"/>
+
     <!--  State machine closed for construction-->
-    <include file="$(find state_machine)/launch/state_machine.launch"/>
```
Comment (flamma7) [inline cusub_cortex/cusub_cortex_bringup/launch/cortex.launch:6]: Not correct state machine launch file, but I fix it in my latest pull. I like your idea of adding the config_filename as a parameter

Comment (flamma7) [reply cusub_cortex/cusub_cortex_bringup/launch/cortex.launch:6]: Oh I see you changed it from smach.lauch -> state_machine.launch ....

**Review by flamma7 — APPROVED** [review]
I guess I haven't been up to date on where we want to do namespacing. Clearly the robot name must be in the launch file, but where should the pkg name be namespaced? In the code or in the launch file. In a few of the scripts you namespace inside of the script rosppy.Subscriber('cusub_cortex/....')

File: cusub_cortex/cusub_cortex_bringup/launch/cortex.launch (hunk 1 above)
Comment (DrYerzinia) [reply cusub_cortex/cusub_cortex_bringup/launch/cortex.launch:6]: yea, for the name-spacing, I just kinda name spaced it where it was included or as necessary to make it work.

**Comment by belgiumkansas** [comment]
> I guess I haven't been up to date on where we want to do namespacing. Clearly the robot name must be in the launch file, but where should the pkg name be namespaced? In the code or in the launch file. In a few of the scripts you namespace inside of the script rosppy.Subscriber('cusub_cortex/....')

well that's the issue. when its declared within the topic its not namespacing. namespacing is only inherited from the node. So declaring the topic names as  `<meta-package>/<package>/topics` is the only reliable way to be able to do relative namespacing with multiple subs. Because if the `<meta-package>` and `<package>` are namespaced from the launch they can only do relative referencing within that space.   which is a problem for subscriber packages since they need to do relative referencing outside their `<sub>/<meta-package>/<package>` namespace. 

So they way I went about this was to have all nodes that reference topics outside their "pseudo" namespace only have the `<sub>` namespace and the rest of the "fake" namespacing within this topic declaration for the pub or sub.

Which i thought I documented but apparently did not. my appologies

**Comment by DrYerzinia** [comment]
I was too lazy to think about how to namespace things at 4 am, if we want them in a different place just change it, I fixed the CV stuff but I only namespaced it to the robot level

**Comment by belgiumkansas** [comment]
Okay. I'll merge this in but leave the branch open.

I think its clear that my method of trying do the best of both worlds was just bad.

Policy being the only namespacing will be from the `<sub>` level. with organizational "psuedo namespacing" `<meta-package/<package>/topics` being enforced within the topic declarations.

**Pull request merged**